packages/uxpin-merge-cli/src/steps/serialization/component/implementation/typescript/component/getComponentName.ts

```diff
@@ -1,8 +1,14 @@
 import { TSSerializationContext } from '../context/getSerializationContext';
 import { getComponentFileName } from './getComponentFileName';
 import { ComponentDeclaration } from './getPropsTypeAndDefaultProps';
+import * as ts from 'typescript'
 
 export function getComponentName(context:TSSerializationContext, component:ComponentDeclaration):string {
-  return component.name && component.name.getText()
-    || getComponentFileName(context);
+  if(component.name) {
```

Comment:
Small style fix, there's a space missing here

```suggestion
  if (component.name) {
```